.pipelines/containerSourceData/postgres/Dockerfile-Postgres

```diff
@@ -27,12 +27,6 @@ FROM $BASE_IMAGE
 
 @INCLUDE_MAIN_RUN_INSTRUCTION@
 
-RUN set -eux; \
```

Comment:
the explicit gid/uid may be important here, especially if it mirrors the reference container.

Reply:
After doing more research, I believe that the primary issue for the container build failure stems from the change made in this PR: https://github.com/microsoft/azurelinux/pull/9906. I will revert the above change to the dockerfile so that we can merge just the distroless container change.